src/pages/EnablePayments/AdditionalDetailsStep.js

```diff
@@ -391,10 +391,11 @@ class AdditionalDetailsStep extends React.Component {
                                 onFixTheErrorsLinkPressed={() => {
                                     this.form.scrollTo({y: 0, animated: true});
                                 }}
-                                message={this.props.walletAdditionalDetails.additionalErrorMessage}
+                                message={errorMessage}
                                 isLoading={this.props.walletAdditionalDetails.loading}
```

Comment:
```suggestion
                                isLoading={this.props.walletAdditionalDetails.isLoading}
```

Reply:
Ah, good catch. Merging main made me lose the change!